travel/travel.py

```diff
@@ -25,6 +25,17 @@
 from .res_config import get_basic_passenger_limit
 
 
+def _get_travel_states(self, cr, uid, ids=None, context=None):
```

Comment:
Include a reference that this is due to a bug, and even better if there's an open issue about it and it's linked here